docs/modules/pipelines/pages/job-monitoring.adoc

```diff
@@ -0,0 +1,52 @@
+= Monitoring Jobs
+:description: To ensure the reliability of streaming jobs and their workloads, try implementing some basic monitoring. With Hazelcast, you can monitor the status of streaming jobs to quickly identify job failures for troubleshooting.
+
+{description}
+
+== Monitoring for Status Changes
+
+There are two ways to monitor for job status changes. You can either poll the `Job.getStatus()` method separately for each job or add a `JobStatusListener` to your jobs.
+
+== Using a JobStatusListener
+
+The `JobStatusListener` has several advantages over polling for the job status. 
+
+- You can write custom code for execution on status change. For example, triggering an alert or writing the changed status to a dashboard.
+
+- You can identify jobs in an error state. By default, xref:troubleshoot:error-handling.adoc#processing-guarantees[all streaming jobs are suspended on failure]. When a job moves to the suspended status, the listener is passed a suspension cause; either an error description or requested by a user. 
+
+- You can also work out which were jobs canceled by a user, by calling the `isUserCancelled();` method. This method returns `true` if the user stopped the job, even though the job status is `FAILED`. 
```

Comment:
IMO we should not mix description of `JobStatusListener`/`JobStatusEvent` with regular `Job` API. `JobStatusEvent` contains a limited set of information, additional can be requested using `Job` methods, but this could be described separately (maybe it is already documented) to not cause confusion.

`isUserCancelled()` is in Job. It is a bit similar to `isUserRequested()` in JobStatusEvent, but not the same.

Reply:
I have created a separate section `Monitoring for canceled jobs`